app/forms/collection_form.rb

```diff
@@ -3,91 +3,71 @@
 # Form for a Collection
 class CollectionForm < ApplicationForm
   accepts_nested_attributes_for :related_links, :contact_emails, :managers, :depositors, :reviewers
-  before_validation do
-    blank_managers = managers_attributes.select(&:empty?)
-    next if blank_managers.empty?
-
-    self.managers_attributes = managers_attributes - blank_managers
-  end
-  validates :managers_attributes, length: { minimum: 1, message: 'must have at least one manager' } # rubocop:disable Rails/I18nLocaleTexts
 
   def self.immutable_attributes
     ['druid']
   end
 
+  attribute :access, :string, default: 'world'
+  attribute :custom_rights_statement_instructions, :string
+  attribute :custom_rights_statement_option, :string, default: 'no'
+  attribute :default_license, :string
+  attribute :deposits_contact_email, :string
```

Comment:
Here's the new attribute. I chose this option over mucking about with the existing nested attributes (`contact_emails_attributes`).

Reply:
Why isn't this just another nested attribute?

Reply:
Why would it be a nested attr? The design seems to indicate that this is a single, simple field.

Reply:
Oh, I totally missed that. Reconsidering.